README.md

```diff
@@ -10,9 +10,31 @@ Install via the [AUR](https://aur.archlinux.org/packages/downgrade/).
 
 ## Usage
 
-Downgrade some packages, checking both local cache and the [A.L.A.][ala]:
+Below is the usage script for `downgrade`:
 
-[ala]: https://wiki.archlinux.org/index.php/Arch_Linux_Archive
+```
+Usage: downgrade [option...] <pkg> [pkg...] [-- pacman_option...]
+
+  Options:
+    --pacman <command>
+                    pacman command to use, defaults to "pacman"
+    --pacman-conf <file-path>
+                    pacman configuration file, defaults to "/etc/pacman.conf"
+    --arch <architecture>
+                    target architecture, defaults to output of "uname -m"
+    --ala-url <url>
+                    location of ALA server, defaults to "https://archive.archlinux.org"
+    --ala-only      only use ALA server
+    --cached-only   only use cached packages
+    --nosudo        do not use sudo even when available, use su
+    -h, --help      show help script
+
+  Note:
+    Any options after the -- character sequence will be treated as pacman options.
+    See downgrade(8) for further details.
+```
+
+Downgrade packages checking both local cache and the [A.L.A.](https://wiki.archlinux.org/index.php/Arch_Linux_Archive), which is the default behavior.
```

Comment:
:art: can you hard-warp at 80 columns throughout all the markdown?

We should enable `prettier-markdown` in `.restyled.yaml`.

Reply:
Updated `.restyled.yaml`, the hard-wrapping done too.